docs/reference/accounts.md

```diff
@@ -0,0 +1,120 @@
+# Accounts
+
+TigerBeetle uses the same data structures internally and
+externally. This means that sometimes you need to set temporary values
+for fields that TigerBeetle, not you (the user), are responsible.
+
+### Updates
+
+Account fields *cannot be changed by the user* after
+creation. However, debits and credits fields are updated by
+TigerBeetle as transfers happen to and from the account.
+
+## Fields
+
+### `id`
+
+This is the identifier, the primary key, of the account.
+
+As an example, you might use a UUID (encoded as an integer) that ties
+the account back to a row in a SQL database.
+
+Constraints:
+
+* Type is 128-bit unsigned integer (16 bytes)
+* Must be unique
```

Comment:
`id` can't be zero or `intMax(u128)`.

Reply:
Is `intMax(u128)` the same as `2^128 - 1`. I'm trying to leave zig-specific stuff out of any non-zig docs so that folks unfamiliar with Zig have an easier time.

Reply:
Good point! Yes, it is.